src/content/docs/en/guides/server-side-rendering.mdx

```diff
@@ -5,14 +5,24 @@ i18nReady: true
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 import Since from '~/components/Since.astro';
 
+Server-side rendering (SSR) refers to generating HTML pages on the server and sending them to the client, e.g., a web browser. 
+
+You may consider enabling server-side rendering in your Astro project if you need the following:
+
+- **Using API endpoints**: SSR enables you to create specific pages that function as API endpoints for tasks like database access, authentication, and verification while keeping sensitive data hidden from the client.  
+
+- **Creating protected pages**: If you need to restrict access to a page based on user privileges, you can enable SSR to handle user access on the server.  
+
+- **Frequently changing content**: if a page's content changes frequently, such as through a content management system, enabling SSR can generate these pages without requiring a static rebuild every time.
+
+## Enabling SSR in Your Project
+
 **Server-side Rendering**, aka SSR, can be enabled in Astro. When you enable SSR you can:
```

Comment:
```suggestion
**Server-side Rendering**, can be enabled in Astro. When you enable SSR you can:
```

Do we still need to address 'aka SSR' here, since its now explained on line 8?